Cognite.Simulator.Utils/SimulationRunnerBase.cs

```diff
@@ -439,25 +442,39 @@ async void PublishSimulationRunStatus(string runStatus, CancellationToken token)
                     configObj,
                     validationEnd);
             }
-            // Run the simulation
-            await RunSimulation(
-                simEv,
-                startTime,
-                modelState,
-                configState,
-                configObj,
-                samplingRange,
-                token).ConfigureAwait(false);
+            try
```

Comment:
we should have the try-catch block nested into `using (LogContext.PushProperty("LogId"` (bigger), so we can log this entire `InitSimulationRun` and not just `RunSimulation` bit